### ISSUE TITLE
Overblocking Nimiq related applications

### ISSUE THREAD

**Issue by mar-v-in** [xd4rker/MinerBlock#31]
I noticed your extension filters (which were just embedded by EasyPrivacy) is overblocking some Nimiq related applications.

The relevant entries are
- `nimiq.watch`
- `nimiqtest.net`

`nimiq.watch` is a Nimiq block explorer service. It provides an API under `api.nimiq.watch` domain that is used by third-parties to access its services. There is also a mining pool hosted under `pool.nimiq.watch`, which you might want to continue to block (as it might be used by mining clients embedded into third-party websites, although I doubt it), but should not cause the api service to be blocked.

`nimiqtest.net` is the host name of some network nodes in the nimiq testnet network. These nodes do provide all the information that can be gathered from the testnet, but not any logic for mining. Although some miners might optionally connect to these hosts, it is not a requirement, so it makes no sense to block the host to block mining. However some client web applications, most notably https://safe.nimiq-testnet.com/, the standard client to access the nimiq testnet, require access to these hosts.

Please remove these two filters (or at least for the case of `nimiq.watch`, update it) to no longer overblock Nimiq web apps.

**Comment by xd4rker** [comment]
Hi, sorry if it's interfering with Nimiq's related apps. Does adding those fix the issue ?

```
*://cdn.nimiq.com/nimiq.js*
wss://pool.nimiq.watch/*
wss://node.nimiqpool.com/*
wss://pool.porkypool.com/*
wss://pool.nimiqchain.info/*
wss://eu.nimpool.io/*
wss://node.philpool.com/*
wss://eu.sushipool.com/*
```

**Comment by mar-v-in** [comment]
`*://cdn.nimiq.com/nimiq.js*` is definitely causing even more applications to break: These files connect the logic to connect to the p2p network. The other filters look ok to me.

**Comment by xd4rker** [comment]
Closing this as it has been fixed in 9037ab35580811bfb13a8a46cabfb8923f979c61. Please feel free to re-open if necessary.
